supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -128,4 +128,30 @@ Avoid long introductions and unnecessary context. Shorten unnecessarily long sen
 === Principle 5: Error recovery, verification, and troubleshooting
 Recognize that people make mistakes and need to verify that they have completed a task. Be sure to include troubleshooting, error recovery, and verification steps.
 
+[[users]]
+== Users
+In most cases, the word "user" refers to a person or a person's user account, and therefore would be considered animate. In these cases, use animate personal pronouns such as "who".
```

Comment:
I see "person's user account" mentioned in the animate and inanimate section, so maybe "person's user account" needs to be dropped from the animate paragraph as this account is really a system account (inanimate).

Reply:
+1 Had the same thought.

Reply:
Good point, changed it to system account below.